src/snowflake/snowpark/session.py

```diff
@@ -1556,6 +1556,19 @@ def use_role(self, role: str) -> None:
         """
         self._use_object(role, "role")
 
+    def use_secondary_roles(self, roles: Literal["all", "none"]) -> None:
```

Comment:
I'm thinking of accepting `None` too if "none" is meaningful.

Reply:
done